news-explorer-frontend/src/components/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,23 @@
+import './SearchForm.css';
+import { useRef } from 'react';
+
+function SearchForm(props) {
+  const keywordInput = useRef();
```

Comment:
**Might be better**


`useRef` is rarely used for `inputs`, because it’s preferable to use `controlled` components with `useState`, `value` and `onChange`
To read more about `controlled components`: https://reactjs.org/docs/forms.html#controlled-components

Only when it's not possible to control an input with `onChange` and can't get `value` (in the input type=`file`) and when you need to `focus` an input when you open a popup or get to a page with a form.
To read more: https://reactjs.org/docs/uncontrolled-components.html#the-file-input-tag